Tools/DataHandler.py

```
@@ -212,4 +271,27 @@
         return (universe
                 .IncludeWeeklys()
                 .Strikes(-self.strategy.nStrikesLeft, self.strategy.nStrikesRight)
-                .Expiration(max(0, self.strategy.dte - self.strategy.dteWindow), max(0, self.strategy.dte)))
+                .Expiration(max(0, self.strategy.dte - self.strategy.dteWindow), max(0, self.strategy.dte)))
+
+    def _initializeGreeks(self, security):
+        """Initialize Greeks for an option contract after it's been added"""
+        try:
+            if security.Type in [SecurityType.Option, SecurityType.IndexOption]:
```

Ruff (0.8.0)

`SecurityType` may be undefined, or defined from star imports

(F405)